internal/testing/ticket_generator.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package testing provides testing primitives for the codebase.
+package testing
+
+import (
+	"fmt"
+
+	structpb "github.com/golang/protobuf/ptypes/struct"
+	"open-match.dev/open-match/pkg/pb"
+)
+
+// PropertyManifest defines the required fields that we need to generate tickets for testing.
+type PropertyManifest struct {
```

Comment:
Simply Property - or PropertyData? Manifest may not be the right term for what this stores.